sovtoken/sovtoken/test/helpers/helper_node.py

```diff
@@ -33,17 +37,17 @@ def get_last_ledger_transaction_on_nodes(self, ledger_id):
     @property
     def xfer_handler(self):
         """ Get the xfer request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[XFER_PUBLIC][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[XFER_PUBLIC] if isinstance(h, XferHandler))
 
     @property
     def mint_handler(self):
         """ Get the xfer request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[MINT_PUBLIC][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[MINT_PUBLIC] if isinstance(h, MintHandler))
 
     @property
     def nym_handler(self):
         """ Get the NYM request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[NYM][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[NYM] if isinstance(h, NymHandler))
```

Comment:
please, use get_handler_by_type_wm helper method from indy-plenum for this

Reply:
fixed